docs/DEVELOPMENT.md

```diff
@@ -169,7 +169,8 @@ Follow the steps below to publish new operator release to
 
 * Generate OLM catalog for new release
 ``` console
-$ make operator-generate-catalog VERSION=<X.Y.Z> #semantic version number
+$ make operator-clean-catalog
+$ make operator-generate-catalog VERSION=<X.Y.Z> REPLACES=<X.Y.Z> #semantic version numbers
```

Comment:
While you are touching this, can you add "# semantic version numbers without v prefix"?

X.Y.Z implies it, but it's better to be explicit.

Reply:
Done.